packages/react-i18n/src/tests/i18n.test.ts

```diff
@@ -784,16 +784,48 @@ describe('I18n', () => {
   });
 
   describe('#formatCurrency()', () => {
+    const amount = 1234;
     const currency = 'USD';
 
-    it('formats the number as a currency', () => {
+    it('formats short when there is no currency in the options', () => {
```

Comment:
@sunical These tests are copied from the `form:auto` tests just below this - they confirm that not passing the `form` option produces the same behaviour as `form:auto`.